pages/home_page.py

```diff
@@ -5,16 +5,28 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
 from pages.base import Base
 
 
 class Home(Base):
 
+    _groups_link_locator = (By.PARTIAL_LINK_TEXT, 'groups')
+    _functional_areas_link_locator = (By.PARTIAL_LINK_TEXT, 'functional areas')
```

Comment:
It's best practice to use CSS selectors and avoid locating by link text, as the latter is vague and will break under different locales.